core/domain/exp_services.py

```diff
@@ -1795,3 +1799,203 @@ def save_multi_exploration_math_rich_text_info_model(
+    if old_state_dict['interaction']['customization_args'] != (
+            new_state_dict['interaction']['customization_args']):
+        property_name_to_new_value_mapping.append(
+            (
+                exp_domain.STATE_PROPERTY_INTERACTION_CUST_ARGS,
+                new_state_dict['interaction']['customization_args']))
+    if old_state_dict['content'] != (
+            new_state_dict['content']):
+        property_name_to_new_value_mapping.append(
+            (exp_domain.STATE_PROPERTY_CONTENT, new_state_dict['content']))
+
+    if old_state_dict['written_translations'] != (
+            new_state_dict['written_translations']):
+        property_name_to_new_value_mapping.append(
+            (
+                exp_domain.STATE_PROPERTY_WRITTEN_TRANSLATIONS,
+                new_state_dict['written_translations']))
+    if old_state_dict['interaction']['default_outcome'] != (
+            new_state_dict['interaction']['default_outcome']):
+        property_name_to_new_value_mapping.append(
+            (
+                exp_domain.STATE_PROPERTY_INTERACTION_DEFAULT_OUTCOME,
+                new_state_dict['interaction']['default_outcome']))
+    if old_state_dict['interaction']['hints'] != (
+            new_state_dict['interaction']['hints']):
+        property_name_to_new_value_mapping.append(
+            (
+                exp_domain.STATE_PROPERTY_INTERACTION_HINTS,
+                new_state_dict['interaction']['hints']))
+    if old_state_dict['interaction']['solution'] != (
+            new_state_dict['interaction']['solution']):
+        property_name_to_new_value_mapping.append(
+            (
+                exp_domain.STATE_PROPERTY_INTERACTION_SOLUTION,
+                new_state_dict['interaction']['solution']))
+    if old_state_dict['interaction']['answer_groups'] != (
+            new_state_dict['interaction']['answer_groups']):
+        property_name_to_new_value_mapping.append(
+            (
+                exp_domain.STATE_PROPERTY_INTERACTION_ANSWER_GROUPS,
+                new_state_dict['interaction']['answer_groups']))
+
+    for property_name, new_value in property_name_to_new_value_mapping:
+        change_lists.append(
+            exp_domain.ExplorationChange({
+                'cmd': exp_domain.CMD_EDIT_STATE_PROPERTY,
+                'state_name': state_name,
+                'property_name': property_name,
+                'new_value': new_value
+            }))
+    return change_lists
+
+
+def get_latex_strings_and_exp_ids_for_generating_svgs():
+    """Returns a batch of LaTeX strings from explorations which have LaTeX
+    strings without SVGs.
+
+    Returns:
+        dict(str, list). The dict having each key as an exp_id and value
+        as a list of LaTeX string.
+    """
+
+    latex_strings_mapping = {}
+    exploration_math_rich_text_info_models = (
+        exp_models.ExplorationMathRichTextInfoModel.get_all())
+    size_of_svgs_in_batch = 0
```

Comment:
Always include units in your variable names. (I think this has been mentioned in previous reviews.)

Reply:
done